### ISSUE TITLE
`no-abusive-eslint-disable` false positives 

### ISSUE THREAD

**Issue by SamuelT-Beslogic** [sindresorhus/eslint-plugin-unicorn#2162]
See example below,
the comment in my example is meant to explain why all the eslint-disables in a file w/o having to explain it for every line. It is picked up by `unicorn/no-abusive-eslint-disable` even though it doesn't actually abusively disable eslint.

![image](https://github.com/sindresorhus/eslint-plugin-unicorn/assets/84519004/a38b0065-b460-4fd0-8588-12c9a08ac2e3)
```js
// eslint-disables: This file is a polyfill, it's meant to abstract away rules
const x = ''
```

Workaround: prepend the comment with something like `NOTE: `
